vcd_cli/right.py

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright (c) 2017 VMware, Inc. All Rights Reserved.
+#
+# This product is licensed to you under the
+# Apache License, Version 2.0 (the "License").
+# You may not use this product except in compliance with the License.
+#
+# This product may include a number of subcomponents with
+# separate copyright notices and license terms. Your use of the source
+# code for the these subcomponents is subject to the terms and
+# conditions of the subcomponent's license, as noted in the LICENSE file.
+#
+import click
+
+from pyvcloud.vcd.org import Org
+
+from vcd_cli.utils import restore_session
+from vcd_cli.utils import stderr
+from vcd_cli.utils import stdout
+from vcd_cli.vcd import vcd
+
+
+@vcd.group(short_help='work with rights')
+@click.pass_context
+def right(ctx):
+    """Work with rights
+
+\b
+    Examples
+        vcd right list -o myOrg
+            Get list of rights in the specified organization 
+            (defaults to current organization in use).
+    """  # NOQA
+    if ctx.invoked_subcommand is not None:
+        try:
+            restore_session(ctx)
+        except Exception as e:
+            stderr(e, ctx)
+
+
+@right.command('list', short_help='list rights in the specified or current org')
+@click.pass_context
+@click.option('-o',
+              '--org',
+              'org_name',
+              required=False,
+              metavar='<org-name>',
+              help='name of the org')
+def list_rights(ctx, org_name):
+    try:
+        client = ctx.obj['client']
+        if org_name is None:
+            org_name = ctx.obj['profiles'].get('org_in_use')
+        org = Org(client, resource=client.get_org_by_name(org_name))
```

Comment:
If `org-name` is omitted and will use the `org_in_use`, a better approach is to use `['profiles'].get(org_href)' that holds the href of the org_in_use. On a separate task, we should rename `org_href` to `org_in_use_href` and `vcd_href` to `vcd_in_use_href` to avoid any confusion.

Reply:
I will add a separate task for the above request as there are multiple places I need to change the same.

Reply:
If i am not too late, the command _vcd right list_ should instead be _vcd org list_rights_, we are asking for rights in an org, so asking the question to org makes more sense.